types/2022-08-01/index.d.ts

```diff
@@ -281,27 +279,6 @@ declare module 'stripe' {
       event: 'response',
       handler: (event: Stripe.ResponseEvent) => void
     ): void;
-
-    setProtocol(protocol: string): void;
```

Comment:
this wasn't deprecated here.

Reply:
but we are removing everything else so not sure if keeping the protocol around is worth it.

Reply:
We never shipped definitions from that file, right?

Reply:
Oh sorry no we didn't. I wonder why setProtocol wasn't marked deprecated in [this commit](https://github.com/stripe/stripe-node/commit/5c5b557bcb69eec2e786b652c0c16bea17626355) but the rest were...

Reply:
I assume we just missed it.